DESIGN.md

```diff
@@ -113,6 +114,19 @@ Submit a contribution to the active survey. Anyone can do this while contributio
 }
 ```
 
+### `POST /:dao/nominate`
+
+Nominate a contribution to the active survey. Any DAO member can do this while ratings are being accepted.
+
+#### Request
+
+```ts
+{
+  contributor: string
```

Comment:
Is this an address? A handle? Ideally we'd have both...

Reply:
This is a public key. In line with all the other identity fields (`rater` and `contributor`), entities are only represented by their public keys. We could add `PublicKey` suffixes to all of them, but that might be unnecessarily verbose, and I kinda like the 1:1 mapping of "user is not different from public key".